src/main/java/seedu/address/model/house/Landed.java

```diff
@@ -15,7 +20,60 @@ public class Landed extends House {
      * @param street     The street of the house.
      */
     public Landed(UnitNumber unitNumber, PostalCode postalCode, Street street) {
-        super(unitNumber, postalCode, street);
+        this.postalCode = postalCode;
+        this.street = street;
+        this.unitNumber = unitNumber;
+    }
+
+    /**
+     * Retrieves the unit number of the landed house.
+     *
+     * @return The unit number of the landed house.
+     */
+    @Override
+    public UnitNumber getUnitNumber() {
+        return this.unitNumber;
+    }
+
+    /**
+     * Retrieves the postal code of the landed house.
+     *
+     * @return The postal code of the landed house.
+     */
+    @Override
+    public PostalCode getPostalCode() {
+        return this.postalCode;
+    }
+
+    /**
+     * Retrieves the street of the landed house.
+     *
+     * @return The street of the landed house.
+     */
+    @Override
+    public Street getStreet() {
+        return this.street;
+    }
+
+    /**
+     * Checks if this house is equal to another object.
+     *
+     * @param other The object to compare to.
+     * @return True if the objects are equal, false otherwise.
+     */
+
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof House)) {
```

Comment:
I think this should be ``if (!(other instanceof Landed)) {``

Reply:
Changes have been noted and implemented